bugsnag-benchmarks/src/androidTest/java/com/bugsnag/android/benchmark/JournalBenchmarkTest.kt

```diff
@@ -0,0 +1,50 @@
+package com.bugsnag.android.benchmark
+
+import android.app.Application
+import android.content.Context
+import androidx.benchmark.junit4.BenchmarkRule
+import androidx.benchmark.junit4.measureRepeated
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.bugsnag.android.*
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.ByteArrayOutputStream
+
+@RunWith(AndroidJUnit4::class)
+class JournalBenchmarkTest {
+    @get:Rule
+    val benchmarkRule = BenchmarkRule()
+
+    lateinit var client: Client
+    lateinit var ctx: Context
+
+    @Before
+    fun setUp() {
+        ctx = ApplicationProvider.getApplicationContext<Application>()
+        client = generateClient(ctx)
+    }
+
+    @Test
+    fun serializeEventPayload() {
```

Comment:
This takes ~12,786 ns according to the CI benchmark - how does that roughly compare against the previous solution? It'd be interesting to see what the results are like when I/O is involved.

Reply:
The previous solution was around 40k.